Source/Scene/Expression.js

```diff
@@ -52,6 +52,21 @@ define([
         setEvaluateFunction(this);
     }
 
+    function replaceVariables(expression) {
```

Comment:
I'm the wrong perhaps to ask, but I suspect a one-liner regex could do this.  @tfili would know.